### PR TITLE
add simplify from gofmt -s

### PR DESCRIPTION
**Pull request by mjibson**

### CODE REVIEW

**Comment by rytaft** [comment]
Did you still want to make this change?

**Comment by mjibson** [comment]
I did! Was waiting for a reviewer.

File: main.go [hunk 1]
```diff
@@ -37,7 +39,7 @@ var (
 	wrap         = flag.Int("wrap", 100, "column to wrap at")
 	tab          = flag.Int("tab", 8, "tab width for column calculations")
 	overwrite    = flag.Bool("w", false, "overwrite modified files")
-	fast         = flag.Bool("fast", false, "skip running goimports")
+	fast         = flag.Bool("fast", false, "skip running goimports and simplify")
```
Comment (rytaft) [inline main.go:42]: Can you also update the new usage info in the README?

Comment (mjibson) [reply main.go:42]: done

**Pull request merged**